xml/signals/SPTCO-1930/appearance-DW-2-head.xml

```diff
@@ -114,22 +114,27 @@
     <aspectMapping>
       <advancedAspect>Diverging Clear</advancedAspect>
       <ourAspect>Clear</ourAspect>
+      <ourAspect>Diverging Clear</ourAspect>
```

Comment:
For SPTCO-1930, the blue mast will be Clear for any yellow mast aspect except for Stop.  If the yellow mast is Stop, then the blue mast will be Approach.

Here is how the masts look without the correction.
![screen shot 2017-05-27 at 11 25 03 am](https://cloud.githubusercontent.com/assets/17280992/26522914/bf4a70e8-42d0-11e7-8bde-8fe844c98d2a.png)
The ourAspect mast is correctly showing a diverging indication since the advancedAspect mast is showing Clear.

![screen shot 2017-05-27 at 11 25 20 am](https://cloud.githubusercontent.com/assets/17280992/26522916/ccfc964e-42d0-11e7-9532-c1899dedcc9b.png)
Now it is not showing a diverging indication.  The ourAspect mast has defaulted to Clear.

Maybe there is a terminology problem.  I read advancedAspect as **next** aspect.  
`if the next aspect is x then my aspect should be one the following ourAspect entries based on route, speed, etc.`


Dave Sand